src/homework4/task2.py

```diff
@@ -0,0 +1,20 @@
+"""Дан список стран и городов каждой страны. Затем даны названия городов.
+   Для каждого города укажите, в какой стране он находится.
+   Входные данные:
+   Программа получает на вход количество стран N. Далее идет N строк,
+   каждая строка начинается с названия страны, затем идут названия городов
+   этой страны. В следующей строке записано число M, далее идут M запросов
+   — названия каких-то M городов, перечисленных выше.
+   Выходные данные:
+   Для каждого из запроса выведите название страны, в котором находится
+   данный город
+"""
+
+countries = {}
+
+for data in range(int(input())):
```

Comment:
overall good solution but it would show wrong result for Brest in France and Belarus

Reply:
not addressed